src/main/java/com/cottonfactory/products/controllers/SkirtsController.java

```diff
@@ -0,0 +1,23 @@
+package com.cottonfactory.products.controllers;
+
+import com.cottonfactory.products.entities.Skirt;
+import com.cottonfactory.products.repositories.SkirtsRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/products/")
+public class SkirtsController {
```

Comment:
1 teeny tiny update needed: add the `@Api` annotation to the controller. See the README for details!

Reply:
done